core/server/api/v2/utils/serializers/output/slugs.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:slugs');
+const common = require('../../../../../lib/common');
+
+module.exports = {
+    all(slug, apiConfig, frame) {
+        debug('all');
+        if (!slug) {
```

Comment:
🤔 This is not really output serialization. I think i would put it to the controller.
Long-Term: Model layer should straight throw an error and not return `null?

Reply:
Yeah agree ideally Model layer should directly take care of it, avoided putting it controller initially as thought to keep it min, but yeah makes more sense there for now, will update 👍

Reply:
> Yeah agree ideally Model layer should directly take care of it, avoided putting it controller initially

Yeah i understand :)